week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -42,20 +42,58 @@ My sources say no.
 Outlook not so good.
 Very doubtful.
 */
+let possibleOutcome = ["It is certain.", //very positive
+  "It is decidedly so.", 
+  "Without a doubt.",
+  "Yes - definitely.",
+  "You may rely on it.",
+
+  "As I see it, yes.", //positive
+  "Most likely.",
+  "Outlook good.",
+  "Yes.",
+  "Signs point to yes.",
+
+  "Reply hazy, try again.", //negative
+  "Ask again later.",
+  "Better not tell you now.",
+  "Cannot predict now.",
+  "Concentrate and ask again.",
+
+  "Don't count on it.", //very negative
+  "My reply is no.",
+  "My sources say no.",
+  "Outlook not so good.",
+  "Very doubtful."
+]
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
+  let outcome = possibleOutcome[Math.floor(Math.random() * possibleOutcome.length)];
+  console.log('The ball has shaken!')
+  return outcome
+}
 
 // The answer should come from shaking the ball
-let answer;
-
+let answer = shakeBall;
 // When checking the answer, we should tell someone if the answer is
 // - very positive
 // - positive
 // - negative
 // - very negative
-function checkAnswer() {}
+function checkAnswer() {
+  let indexValue = possibleOutcome.indexOf(answer);
+  if (indexValue <= 4) {
+  return `very positive`}
+ else if  (indexValue > 4 & indexValue <=9){
+   return `positive`}
+else if (indexValue > 9 & indexValue <= 14) {
+  return `negative`}
+  else {
+    return `very negative`
+  }
+}
```

Comment:
Nice work, the only suggestion I have is to format it to make it more readable. If you wish to do it automatically the shortcut should be Alt+Shift+F on Windows or Ctrl+Shift+I on Linux